pkg/yurttunnel/handlerwrapper/tracerequest/tracereq.go

```diff
@@ -17,29 +17,58 @@ limitations under the License.
 package tracerequest
 
 import (
+	"errors"
+	"net"
 	"net/http"
 	"time"
 
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/informers"
+	corelisters "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog/v2"
 
 	hw "github.com/openyurtio/openyurt/pkg/yurttunnel/handlerwrapper"
 	"github.com/openyurtio/openyurt/pkg/yurttunnel/server/metrics"
 )
 
+const (
+	proxyHostHeaderKey = "X-Tunnel-Proxy-Host"
```

Comment:
I find that we refer to this in many places. Maybe we should put it into `constants.go`?

Reply:
fixed